README.md

```diff
@@ -33,16 +34,14 @@ jobs:
   security:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-    - name: Run Snyk to check for vulnerabilities
-      uses: snyk/actions/node@master
-      env:
-        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      - uses: actions/checkout@master
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/node@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
 ```
 
-
-If you want to send data to Snyk, and be alerted when new vulnerabilities are discovered, you can run Snyk monitor like so:
-
+If you want to send data to Snyk, and be alerted when new vulnerabilities are discovered, you can run [Snyk monitor](https://support.snyk.io/hc/en-us/articles/360000920818-What-is-the-difference-between-snyk-test-protect-and-monitor-) like so:
```

Comment:
```suggestion
If you want to send data to Snyk, and be alerted when new vulnerabilities are discovered, you can run [`snyk monitor`](https://support.snyk.io/hc/en-us/articles/360000920818-What-is-the-difference-between-snyk-test-protect-and-monitor-) like so:
```

Reply:
I don't like doing that, because GH makes the link almost invisible in code blocks